creed/file_hash_test.py

```diff
@@ -0,0 +1,56 @@
+import os
+import shutil
+import tempfile
+import pytest
+
+import skimage.io as io
+import numpy as np
+
+from creed import file_hash
+
+
+def test_get_hash():
+    with tempfile.TemporaryDirectory() as temp_dir:
+
+        for img in range(2):
+            array = np.random.rand(36).reshape((6, 6))
+            temp_file_path = os.path.join(temp_dir, 'test_file_{}.tiff'.format(img))
+            io.imsave(temp_file_path, array)
+
+        hash1 = file_hash.get_hash(os.path.join(temp_dir, 'test_file_0.tiff'))
+        hash2 = file_hash.get_hash(os.path.join(temp_dir, 'test_file_1.tiff'))
+
+        assert hash1 != hash2
+
+
+def test_compare_directories():
+    with tempfile.TemporaryDirectory() as top_level_dir:
+        dir_1 = os.path.join(top_level_dir, 'dir_1')
+        dir_2 = os.path.join(top_level_dir, 'dir_2')
+        os.makedirs(dir_1)
```

Comment:
I got confused for a second why `dir_2` wasn't created like `dir_1`, but then saw line 39.  Might help readability here to move line 29 to line 38 above the `shutil.copytree` line.